scripts/justinstall.sh

```diff
@@ -0,0 +1,123 @@
+cd $WORKSPACE
+ALTISCALE_RELEASE=${ALTISCALE_RELEASE:-0.1.0}
```

Comment:
Shouldn't the default starts with ADC `4.0.0` or something like that? This is referring to Altiscale release right?